src/Hackport/Command/Update.hs

```diff
@@ -1,16 +1,17 @@
 module Hackport.Command.Update
-  ( updateAction
+  ( CabalInstall.updateAction
   ) where
 
 import Distribution.Simple.Command (CommandUI(..))
```

Comment:
The CI runs with the `pedantic` flag enabled, which turns on `-Werror`. So, you'll have to fix any warnings for the tests to pass.

Reply:
> The CI runs with the `pedantic` flag enabled, which turns on `-Werror`. So, you'll have to fix any warnings for the tests to pass.

at first I need to do something with update command, maybe running CmdUpdate from cabal-install is good way but need to rework arguments used, new cabal install wants some "NixStyle opts" to be passed.